lib/ojs.ml

```diff
@@ -13,7 +13,12 @@
 *)
 
 
-type t
+type t = Jsoo_runtime.Js.t
+
+external pure_js_expr: string -> t = "caml_pure_js_expr"
+let global = pure_js_expr "joo_global_object"
```

Comment:
Do you think it could be a good idea to add this in "jsoo_runtime" ?
And what about null & undefined ?

Reply:
I don't know. maybe @dbuenzli has an opinion.

Reply:
No opinion :-) FWIW I don't expose `pure_js_expr` and here are its usages in `brr`.

```
> git grep pure_js_expr
src/jstr.ml:external jv_pure_js_expr : string -> 'a = "caml_pure_js_expr"
src/jstr.ml:let jv_global = jv_pure_js_expr "globalThis"
src/jstr.ml:let iterator : jv = jv_pure_js_expr "Symbol.iterator"
src/jv.ml:external pure_js_expr : string -> 'a = "caml_pure_js_expr"
src/jv.ml:let null = pure_js_expr "null"
src/jv.ml:let undefined = pure_js_expr "undefined"
src/jv.ml:let global = pure_js_expr "globalThis"
src/jv.ml:let true' = pure_js_expr "true"
src/jv.ml:let false' = pure_js_expr "false"
src/jv.ml:  let symbol : jv = pure_js_expr "Symbol.iterator"
```

It seems at the moment I have no equality in `Jv`'s implementation between `Jv.t` and `Jsoo_runtime.Js.t` so I'd need this to be able to use such values from `Jsoo_runtime` but I don't think it would pose any problem.

Reply:
> It seems at the moment I have no equality in Jv's implementation between Jv.t and Jsoo_runtime.Js.t

If this PR gets merged, I think that adding this equality in jv would improve a bit the "interoperability" between brr and gen_js_api !

Reply:
Well I won't add an equality at the `.mli` level, this makes error messages from the compiler more confusing, verbose and harder to understand for end users for little return most of the time.

I'd only add the equality at the `.ml` level so that I can reexport the values for `null`, `undefined` if they are defined in `Jsoo_runtime.Js.t`. But I can provide `{to,of} convertors in `Jv` if that's desirable.